Traffic and performance (#8044)
umbraco/UmbracoDocs: umbraco-cloud/optimize-and-maintain-your-site/monitor-and-troubleshoot/traffic-and-performance.md

For Cloud projects with dedicated options, or shared plans at other granularities, you see the average assigned private bytes displayed in bytes. The statistics tiles below display the maximum, average, and minimum allocation of private bytes based on the selected granularity.

#### Edge Requests

The chart depicts the number of HTTP requests served through Cloudflare's edge network for the selected hostnames in the selected time range.

The [Traffic Breakdown Tables](#traffic-breakdown-tables) section further down the page shows a more detailed view of edge traffic, broken down by status code, cache status, HTTP version, path, host, and data center.

#### Edge Data Transfer

The chart depicts the amount of data (in bytes) transferred through Cloudflare's edge network for the selected hostnames in the selected time range.

![Edge requests and data transfer](../../.gitbook/assets/tp-graph-edge-requests-and-data-transfer.png)

> The [Traffic Breakdown Tables](#traffic-breakdown-tables) section further down the page shows a more detailed view of edge data transfer, broken down by status code, cache status, HTTP version, path, host, and data center.

Reviewer: Alternatively, create a new for the categories.

```suggestion
The [Traffic Breakdown Tables](#traffic-breakdown-tables) section below details edge data transfer, categorized by status code, cache status, HTTP version, path, host, and data center.
```

#### Paths

The most requested URL paths on your site.

#### Hosts

The hostnames receiving traffic.

#### Data Centers

The Cloudflare data centers serve requests to your site. The data centers help you understand the geographic distribution of your edge traffic.

#### HTTP Methods

The distribution of HTTP methods used by clients (for example, GET, POST, PUT, DELETE).

### Traffic Source

![Traffic source breakdown](../../.gitbook/assets/tp-breakdown-tables-traffic-source.png)

#### Referers

The HTTP referer headers — showing which sites or pages are sending traffic to you.

#### Countries

The geographic origin of requests by country.

#### Source IPs

The client IP addresses that make requests.

#### Source Browsers

The browser types used by visitors (for example, Chrome, Firefox, Safari).

#### Source Device Types

The device classification of visitors — desktop, mobile, tablet, and so on.

#### Source Operating Systems

The operating systems used by visitors (for example, Windows, macOS, Android, iOS).

#### Source User Agents

The full user agent strings from requests.

#### ASNs

> Autonomous System Numbers (ASN) — the network providers from which requests originate.

Reviewer: The acronym need to be defined on the first use, so I've moved it up to the heading instead.

```suggestion
#### Autonomous System Numbers (ASN)

ASNs are the network providers from which requests originate.
```
